src/views/Dashboard/View.vue

```diff
@@ -75,7 +75,7 @@ export default {
       delete localStorage.content
       console.log(localStorage.content)
     } else {
-      fetch("https://speedwagonmailback.herokuapp.com/:id/View/:id")
+      fetch("https://speedwagonmailback.herokuapp.com/email")
```

Comment:
yang ini tu buat baca 1 email aja we, jadi pake /email/find

isi fetchnya juga karna perlu parameter perlu bikin yang mirip ini 
```
fetch("https://speedwagonmailback.herokuapp.com/account/add", {
        method: "POST",
        body: JSON.stringify({
          usern: this.Username,
          pass: this.Password,
          altermail: this.Alt_Email
        }),
        headers: {
          "content-type": "application/json"
        }
      }).then(response => response.json()
      ).then(result => {
        console.log(result)
      })
```